imagetest/test_suites/metadata/setup.go

```diff
@@ -12,40 +12,189 @@ var Name = "metadata"
 
 const (
 	daemonScriptTemplate = `#!/bin/bash
-
-nohup sleep 3600 > /dev/null 2>&1 < /dev/null &
-echo $! > %s
-`
+	nohup sleep 3600 > /dev/null 2>&1 < /dev/null &
```

Comment:
this looks like some reformatting your IDE did, but fyi it means the script will have this indentation. probably you want to restore the previous un-indented string literal for every script you're using backtics for in this file.

Reply:
No, this was intentional. The indentation in the scripts don't cause any problems, and the un-indented strings were significantly harder to read. It was a judgement call of code readability over perfectly formatted scripts in metadata.